### PR TITLE
Add admin button to refresh data

### PR DESCRIPTION
**Pull request by comzeradd**

### CODE REVIEW

**Comment by comzeradd** [comment]
@akatsoulas r?

File: cardano/base/admin.py [hunk 1]
```diff
@@ -1,6 +1,10 @@
-from django.contrib import admin
+from django.contrib import admin, messages
+from django.conf.urls import url
+from django.core.urlresolvers import reverse
```
Comment (akatsoulas) [inline cardano/base/admin.py:3]: Please fix the order of the import files.

File: cardano/base/helpers.py [hunk 2]
```diff
@@ -0,0 +1,32 @@
+from django.conf import settings
```
Comment (akatsoulas) [inline cardano/base/helpers.py]: I would put these in a `utils.py` file - merely a suggestion.

File: cardano/base/helpers.py [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+
+def fetch_countries():
+    mozillians_client = MozilliansClient(settings.MOZILLIANS_API_URL,
+                                         settings.MOZILLIANS_API_KEY)
+    countries = sorted(product_details.get_regions('en-US').values())
+
+    for country in countries:
+        vouched = mozillians_client.get_users(
+            params={
```
Comment (akatsoulas) [inline cardano/base/helpers.py]: It seems that you can avoid a few line breaks here.

File: cardano/base/templates/admin/base/change_list.html [hunk 4]
```diff
@@ -0,0 +1,15 @@
+{% extends "admin/change_list.html" %}
+
+{% block object-tools-items %}
+  <li>
+    <a href="{% url 'admin:refresh_mozillians' %}">
+      Refresh Mozillians
```
Comment (akatsoulas) [inline cardano/base/templates/admin/base/change_list.html]: s/Refresh/Fetch

**Comment by comzeradd** [comment]
@akatsoulas PR updated

**Pull request closed without merging**